Makefile

```diff
@@ -34,3 +34,7 @@ clean:
 integration:
 	make clean && make
 	python reverse_proxy_integration.py
+
+integration_302:
+	make clean && make
+	python reverse_proxy_302_integration.py
```

Comment:
Do these two need to be separate and split into two different headers in the makefile? Or can they both be put under one header? There is no issue with this. I am just curious.